### DISCUSSION TITLE
Interest in a stage-server

### DISCUSSION THREAD

**Discussion by oenu** [Stage-Whisper/Stage-Whisper#26] [Ideas]
Is there interest in allowing an installation of stage whisper to run an webserver that can be connected to by other installations? I am thinking that you may want to have a more powerful machine running transcription for a large group of users.

This would also lend itself to dockerization allowing for self-hosted transcription which would integrate well with Plex users, data ~~hoarders~~ collectors, or archivists

**Comment by oenu** [comment]
As a note on this, if we end up using electron it would be trivial to add a web-client interface for direct control of a remote machine. It would require some work to add a file transfer system to get the file to the host machine, and it wouldnt act as a headless server for multiple clients, but it would be accessible over a network. 

The beauty of tying yourself to web frameworks

**Comment by literallymapping** [comment]
This seems like an obvious feature at some point, like for enterprise clients. I imagine it's valuable, but for a user like me, the local functionality is important for privacy. I'd call this a nice to have and maybe not necessary for an MVP.
